expression/column.go

```diff
@@ -375,3 +375,22 @@ func IndexInfo2Cols(cols []*Column, index *model.IndexInfo) ([]*Column, []int) {
 	}
 	return retCols, lengths
 }
+
+// FindColumnsByUniqueIDs will find columns by checking the unique id.
+func FindColumnsByUniqueIDs(cols []*Column, ids []int) []*Column {
+	retCols := make([]*Column, 0, len(ids))
+	for _, id := range ids {
+		found := false
+		for _, col := range cols {
+			if col.UniqueID == id {
+				retCols = append(retCols, col)
+				found = true
+				break
+			}
+		}
+		if !found {
+			break
```

Comment:
This check can be removed. we can make this function to find all the columns in the unique id list.

Reply:
But we still need this check. In such case, we need to return nil or a full slice?

Reply:
The function caller can check whether there is any column can not be found in `cols` by:

```go
len(ids) == len(retCols)
```

Reply:
Another consideration is, this function is only used in the test currently, all the column identified by `ids` can be found in `cols`. We can extend this function in the future when we meet more complicated scenarios. For now, I think we'd better to keep it simple.